integration-tests/features/steps/stack_analysis.py

```diff
@@ -199,49 +199,32 @@ def check_stack_analyses_response(context, url):
     jsonschema.validate(resp_json, schema)
 
 
-@when('I download and parse outlier probability threshold value')
-def download_and_parse_outlier_probability_threshold_value(context):
-    """Download and parse outlier probability threshold value.
+def check_frequency_count(usage_outliers, package_name):
+    """Check the frequency count attribute.
 
-    This Special step that is needed to get the stack analysis outlier
-    probability threshold.
+    Try to find frequency count value for given package and check that
+    the value is within permitted range.
     """
-    content = download_file_from_url(STACK_ANALYSIS_CONSTANT_FILE_URL)
-    context.outlier_probability_threshold = parse_float_value_from_text_stream(
-        content, STACK_ANALYSIS_OUTLIER_PROBABILITY_CONSTANT_NAME)
+    # TODO: uncomment this after the https://github.com/openshiftio/openshift.io/issues/843
```

Comment:
I think we can remove this, since we will resolve that issue by merging this.

Reply:
well it depends :) if we merge this PR first, the tests will fail for other master builds, so people won't be happy.

Reply:
Ah I get it. I've deleted the comment and the 'fallback' condition.